crates/core/src/prelude/src/index.ts

```diff
@@ -0,0 +1,330 @@
+import "core-js/actual/url";
+import "core-js/actual/url/to-json";
+import "core-js/actual/url-search-params";
+import { URLPattern } from "urlpattern-polyfill";
+
+declare module globalThis {
+  var URLPattern;
+  function __decodeUtf8BufferToString(...args): string;
+  function __encodeStringToUtf8Buffer(...args): ArrayBufferLike;
+  var __getTime;
```

Comment:
Do we need to expose these? I believe we actually delete these before we hand off control to the programmer. Is it possible to just add `// @ts-ignore` annotations to their use?

Reply:
yea that's a good idea - I think I just got started trying to type everything before I knew what was actually needed in the globals

Reply:
These are kind of bootstrap functions. We write them in rust and expose to the JS layer, then we wrap them, then we delete their references. At least that's the idea. I can't recall if I actually delete all of them or not. we don't want people to use them directly though.